terraform/production/feature_flags.tf

```diff
@@ -6,3 +6,12 @@ resource "aws_secretsmanager_secret_version" "show_historic_data_feature_flag" {
   secret_id     = aws_secretsmanager_secret.show_historic_data_feature_flag.id
   secret_string = "true"
 }
+
+resource "aws_secretsmanager_secret" "deallocation_date_feature_flag" {
```

Comment:
Please don't enable or run terraform for production for now. This is due to reasons discussed outside this PR

Reply:
^ We'll need to create these manually through the console as like Tuomo said, we can't / don't want to be running Terraform on production.